### DISCUSSION TITLE
Page Builder Issue

### DISCUSSION THREAD

**Discussion by JogenderInfinity** [statamic/cms#8332] [General]
Hi guyz,
I am trying to add a pag_builder in my collection but after adding I am getting this error. Let me know any one can help me to come out this issue. Thanks a lot.
![image](https://github.com/statamic/cms/assets/127183553/37203240-f869-4b24-94a8-95f9ab96f201)

**Comment by BrandonOldenhof** [comment]
Hello! 👋 
It's in the CP which makes me thing you have a repeater/bard editor/grid or something similar that's trying to load a field/component/type called "fields" but it's not there.

From my experience there are 2 things you can try to fix this:

1. Run `php artisan optimize:clear`: this will clear the backend (and frontend!) cache that Statamic uses in the CP that contains, I think, cached versions of the fields' configurations.
2. Go through your pagebuilder's yaml file and verify that all your fields and keys are present and valid. This is much harder to do. I find that removing fieldsets from the pagebuilder's yaml file one-by-one is the easiest way to do this, especially if you're using Git. Important: run `php artisan optimize:clear` each time you delete a layout to make sure the configuration hasn't been cached.

**Comment by JogenderInfinity** [comment]
@BrandonOldenhof thanks. I have checked and find that I have missing some installation dependency. I have tried with new installation and find that all the things are working fine. Thanks for your time.
